modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -1209,6 +1209,103 @@ definition {
 			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU");
 	}
 
+	@description = "COMMERCE-10066. User can only view the allowed quantities that match the minimum, max and multiple quantities of a product"
+	@priority = 5
+	test CanViewOnlyCorrectAllowedQuantitiesAreSelectableForProduct {
```

Comment:
Please specify in the description and test name that the test checks the quantities in the product card